Makefile

```diff
@@ -8,7 +8,7 @@ all: cpp java python
 	java java_test java_clean cpp cpp_test cpp_clean \
 	python python_test
 
-docker: cpp cpp_test
+docker: cpp 
```

Comment:
I'm not against this, but you shouldn't be so miserly with your pull requests. :wink: